### DISCUSSION TITLE
Copilot suspended and no response from support, with only 4 conversation

### DISCUSSION THREAD

**Discussion by dtlnor** [community/community#129709] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

support@githubsupport.com sent me a email saying that I'm abusing the copilot service. But what I actually do is only asking it to generate a enum class from some text, with only 4 conversation. Please review this case, it should not reach the rate limit of copilot.

**Comment by Akash1134** [comment, marked as answer]
Hi @dtlnor ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
